sormas-ui/src/main/java/de/symeda/sormas/ui/importer/DataImporter.java

```diff
@@ -285,7 +285,7 @@ public ImportResult runImport() throws IOException, InvalidColumnException, Inte
 			errorReportCsvWriter.writeNext(columnNames);
 
 			// validate headers
-			if (entityClasses != null && entityClasses.length <= 1 || entityProperties.length <= 1) {
+			if (entityClasses != null && entityClasses.length <= 1 || entityProperties.length < 1) {
```

Comment:
the `entityProperties.length` must not be changed, please find another solution for allowing token import with a single column to work